### DISCUSSION TITLE
部品捜しています。

### DISCUSSION THREAD

**Discussion by Kosaigo** [Yowkees/keyball#143] [Q&A] [answered]
keyball39を組み立て中です。promicroのusb端子がもげました。ピンヘッダーとソケットが無くて困っています。手持ちのものでは厚みが1mm以上厚くなってしまいます。購入先を教えてもらえれば助かります。
　発端は電源ONでpromicroが異常に熱くなり、色々とやってみた結果、No.2のLEDも異常発熱
しておりLEDを交換し、OKとなりました。左用です。
その最中にもげました。従い右が組み立て中断しています。宜しくお願い致します。

**Comment by Yowkees** [comment, marked as answer]
Keyball39に付属している12Pピンヘッダーとピンソケットは「廣杉計器」のサイトにて購入できます。
品番は、
FSS-41035-12
PSS-410153-12
になります。
